### ISSUE TITLE
Version 1.0.2+2 broke links inside PDF's

### ISSUE THREAD

**Issue by ton-An** [endigo/flutter_pdfview#72]
Since the update the app crashes when clicking on a link inside a pdf.

*+* I am a bit confused as the changelog states `Add support tap on link inside the pdf` even though that has been possible before the update, or am I misunderstanding something here?

```
D/AndroidRuntime(29624): Shutting down VM
E/AndroidRuntime(29624): FATAL EXCEPTION: main
E/AndroidRuntime(29624): Process: com.example.sastra_ebooks, PID: 29624
E/AndroidRuntime(29624): android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat= flg=0x10008000 }
E/AndroidRuntime(29624): 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2064)
E/AndroidRuntime(29624): 	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1718)
E/AndroidRuntime(29624): 	at android.app.ContextImpl.startActivity(ContextImpl.java:957)
E/AndroidRuntime(29624): 	at android.content.ContextWrapper.startActivity(ContextWrapper.java:405)
E/AndroidRuntime(29624): 	at android.content.ContextWrapper.startActivity(ContextWrapper.java:405)
E/AndroidRuntime(29624): 	at android.content.ContextWrapper.startActivity(ContextWrapper.java:405)
E/AndroidRuntime(29624): 	at io.endigo.plugins.pdfviewflutter.PDFLinkHandler.handleLinkEvent(PDFLinkHandler.java:24)
E/AndroidRuntime(29624): 	at com.github.barteksc.pdfviewer.listener.Callbacks.callLinkHandler(Callbacks.java:177)
E/AndroidRuntime(29624): 	at com.github.barteksc.pdfviewer.DragPinchManager.checkLinkTapped(DragPinchManager.java:103)
E/AndroidRuntime(29624): 	at com.github.barteksc.pdfviewer.DragPinchManager.onSingleTapConfirmed(DragPinchManager.java:69)
E/AndroidRuntime(29624): 	at android.view.GestureDetector$GestureHandler.handleMessage(GestureDetector.java:323)
E/AndroidRuntime(29624): 	at android.os.Handler.dispatchMessage(Handler.java:107)
E/AndroidRuntime(29624): 	at android.os.Looper.loop(Looper.java:214)
E/AndroidRuntime(29624): 	at android.app.ActivityThread.main(ActivityThread.java:7397)
E/AndroidRuntime(29624): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(29624): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
E/AndroidRuntime(29624): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:920)
I/Process (29624): Sending signal. PID: 29624 SIG: 9
Lost connection to device.
```

**Comment by endigo** [comment]
@ton-An Fixed on 1.0.2+3

**Comment by ton-An** [comment]
top👍
